apps/right-to-rent-check/fields.js

```diff
@@ -52,5 +52,16 @@ module.exports = {
     mixin: 'input-text',
     validate: 'required',
     labelClassName: 'visuallyhidden'
+  },
+  'representative': {
+    mixin: 'radio-group',
+    validate: 'required',
+    legend: {
+      className: 'visuallyhidden'
```

Comment:
yes, this is right.  

The issue wasn't legends but label's, see here
https://github.com/UKHomeOffice/right-to-rent-check/blob/master/apps/right-to-rent-check/fields.js#L54

For legend it's an object but by label it's a property called labelClassName.  There is an inconsistency that we should sort out.  It's same for select's